prime-router/src/main/kotlin/Report.kt

```diff
@@ -855,17 +862,46 @@ class Report : Logging {
                             null
                         }
                     }
-                    it.specimenCollectionMethod = row.getStringOrNull("specimen_collection_method")
-                    it.specimenCollectionSite = row.getStringOrNull("specimen_collection_site")
-                    it.specimenType = row.getStringOrNull("specimen_type")
+                    it.specimenCollectionMethod = row.getStringOrNull("specimen_collection_method_text")
+                    it.specimenCollectionMethodCode = row.getStringOrNull("specimen_collection_method_code")
+                    it.specimenCollectionSite = row.getStringOrNull("specimen_collection_site_text")
+                    it.specimenCollectionSiteCode = row.getStringOrNull("specimen_collection_site_code")
+                    it.specimenType = row.getStringOrNull("specimen_type_name")
+                    it.specimenTypeCode = row.getStringOrNull("specimen_type_code")
+                    it.specimenTypeNormalized = if (it.specimenTypeCode != null) {
+                        metadata.findValueSet("monkeypox/specimen_type")?.toDisplayFromCode(it.specimenTypeCode)
+                    } else {
+                        null
+                    }
```

Comment:
A similar pattern to above. Query the valueset for the display value for the code we're given.